aced_submission/grip_load.py

```diff
@@ -1,10 +1,14 @@
 import os
 import orjson
 import requests
-from typing import List
+import orjson
+from typing import List, Generator
+
 
 GRIP_SERVICE = "local-grip:8201"
```

Comment:
It would be useful to know what the expected url, port,  are from the helm perspective.  Should these always be set by env variables, with reasonable defaults?

Reply:
local-grip is the name of the service of the pod that is deployed in helm. Defined here: https://github.com/ACED-IDP/gen3-helm/blob/ohsu-develop/helm/grip/templates/service.yaml#L4.

If that changes this GRIP_SERVICE var must change